code/modules/clothing/head/misc_special.dm

```diff
@@ -234,3 +234,13 @@
 		if(~initial(user.species.flags) & HYPOTHERMIA_IMMUNE)
 			user.species.flags &= ~HYPOTHERMIA_IMMUNE
 		user.faction = initial(user.faction)
+
+/obj/item/clothing/head/beret/sec/ocelot
+	name = "Ocelot's beret"
+	desc = "Ocelot's signature red beret"
+
+/obj/item/clothing/head/beret/sec/ocelot/OnMobLife(var/mob/living/carbon/human/wearer)
+	var/mob/living/carbon/human/W = wearer
```

Comment:
Since you've written 'var/mob/living/carbon/human/wearer' above, this is not actually needed.

It would have been required if the argument was 'var/mob/living/wearer' or something like that, since mob/living doesn't have the "head" variable